src/main.cpp

```diff
@@ -602,8 +613,18 @@ int main ( int argc, char** argv )
         {
             // Client:
             // actual client object
-            CClient
-                Client ( iPortNumber, iQosNumber, strConnOnStartupAddress, strMIDISetup, bNoAutoJackConnect, strClientName, bMuteMeInPersonalMix );
+            CClient Client ( iPortNumber,
+                             iQosNumber,
+                             strConnOnStartupAddress,
+                             strMIDISetup,
+                             bNoAutoJackConnect,
+                             strClientName,
+#ifdef USE_PORTAUDIO
```

Comment:
I'd drop the `#else` block and have the client not have the parameter except when using PortAudio, rather than have an empty string going nowhere.